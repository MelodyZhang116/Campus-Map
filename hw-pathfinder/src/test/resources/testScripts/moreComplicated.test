# includes a more complicated graph to test whether it can find the shortest path

CreateGraph complexGraph

AddNode complexGraph A
AddNode complexGraph B
AddNode complexGraph C
AddNode complexGraph D
AddNode complexGraph E

AddEdge complexGraph A B 1.0
AddEdge complexGraph B E 100.0
AddEdge complexGraph C D 50.0
AddEdge complexGraph D E 1.0

FindPath complexGraph A E