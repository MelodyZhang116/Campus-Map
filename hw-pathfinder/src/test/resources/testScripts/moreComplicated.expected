# includes a more complicated graph to test whether it can find the shortest path

created graph complexGraph

added node A to complexGraph
added node B to complexGraph
added node C to complexGraph
added node D to complexGraph
added node E to complexGraph

added edge 1.000 from A to B in complexGraph
added edge 100.000 from B to E in complexGraph
added edge 50.000 from C to D in complexGraph
added edge 1.000 from D to E in complexGraph

path from A to E:
C to D with weight 50.000
D to E with weight 2.000
total cost: 52.000
